### ISSUE TITLE
The author's attempt to open source weight, poor effect!!

### ISSUE THREAD

**Issue by dlydly111** [xzz777/SCTNet#35]
Have you tried using the author's open source weight evaluation SCTNer? I evaluated Cityscapes with author weights, MIOU no more than 1 percent and wondered what the problem was??

`2024-07-06 16:16:50,433 - mmseg - INFO - OpenCV num_threads is `12
2024-07-06 16:16:50,445 - mmseg - INFO - Loaded 500 images
2024-07-06 16:16:50,519 - mmseg - INFO - initialize SCTNet with init_cfg {'type': 'Pretrained', 'checkpoint': 'pretrain/SCT-B_Pretrain.pth'}
2024-07-06 16:16:50,519 - mmcv - INFO - load model from: pretrain/SCT-B_Pretrain.pth
2024-07-06 16:16:50,520 - mmcv - INFO - load checkpoint from local path: pretrain/SCT-B_Pretrain.pth
E:\桌面\SCTNet\mmseg\models\losses\cross_entropy_loss.py:235: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average ls over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
2024-07-06 16:16:50,785 - mmseg - INFO - initialize VitGuidanceHead with init_cfg {'type': 'Pretrained', 'checkpoint': 'pretrain/Teacher_SegFormer_B3_city.pth'}
2024-07-06 16:16:50,785 - mmcv - INFO - load model from: pretrain/Teacher_SegFormer_B3_city.pth
2024-07-06 16:16:50,785 - mmcv - INFO - load checkpoint from local path: pretrain/Teacher_SegFormer_B3_city.pth
2024-07-06 16:16:50,918 - mmcv - WARNING - The model and loaded state dict do not match exactly

missing keys in source state_dict: feature_transforms.weight, feature_transforms.bias, convstage4.weight, convstage4.bias, convstage3.weight, convstage3.bias

load checkpoint from local path: pretrain/Teacher_SegFormer_B3_COCO.pth
The model and loaded state dict do not match exactly

"CLASSES" not found in meta, use dataset.CLASSES instead
"PALETTE" not found in meta, use dataset.PALETTE instead
tools/test.py:267: UserWarning: SyncBN is only supported with DDP. To be compatible with DP, we convert SyncBN to BN. Please use dist_train.sh which can avoid this error. 
  warnings.warn(
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 500/500, 8.7 task/s, elapsed: 57s, ETA:     0sper class results:

+---------------+------+-------+
|     Class     | IoU  |  Acc  |
+---------------+------+-------+
|      road     | 0.01 |  1.78 |
|    sidewalk   | 0.73 |  0.89 |
|    building   | 0.0  |  0.0  |
|      wall     | 0.07 |  0.07 |
|     fence     | 1.46 | 84.77 |
|      pole     | 0.04 |  0.04 |
| traffic light | 0.04 |  0.04 |
|  traffic sign | 0.25 |  0.25 |
|   vegetation  | 0.12 |  0.12 |
|    terrain    | 0.01 |  0.01 |
|      sky      | 0.0  |  0.0  |
|     person    | 0.21 |  0.21 |
|     rider     | 0.2  |  0.22 |
|      car      | 0.14 |  0.48 |
|     truck     | 0.0  |  0.0  |
|      bus      | 0.0  |  0.0  |
|     train     | 0.0  |  nan  |
|   motorcycle  | 0.13 |  0.16 |
|    bicycle    | 0.0  |  0.0  |
+---------------+------+-------+
Summary:

+------+------+------+
| aAcc | mIoU | mAcc |
+------+------+------+
| 2.01 | 0.18 | 4.95 |
+------+------+------+
`
